queries/omen/markets.ts

```diff
@@ -85,6 +85,7 @@ const getMarketsQuery = gql`
     $orderDirection: String
     $title_contains_nocase: String
     $creator_in: [String]
+    $category: String
```

Comment:
let's give it the same name: $category_contains

```suggestion
    $category_contains: String
```

Reply:
yeah, makes sense.